lib/indico.rb

```diff
@@ -164,6 +164,13 @@ def self.batch_facial_features(image, config = nil)
     api_handler(preprocess(image, 48, false, false), 'facialfeatures', config)
   end
 
+  def self.facial_localization(image, config = nil)
+    api_handler(preprocess(image, 128, false), 'faciallocalization', config)
+  end
+
+  def self.batch_facial_localization(image, config = nil)
+    api_handler(preprocess(image, 128, true), 'faciallocalization/batch', config)
```

Comment:
Also we change back to `faciallocalization` for the string passed to `api_handler` in `batch_facial_localization`.